client.py

```diff
@@ -4,11 +4,16 @@
 def main():
     client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
     client.connect(('localhost', 50000))
-    client.send("Hello")
-    from_srvr = client.recv(4096)
+
+    data_to_send = ""
+
+    while (data_to_send is not "q"):
```

Comment:
Don't use parentheses in python while/if statements, unless required by multiple conditions

Reply:
fixed